bubo/bot_commands.py

```diff
@@ -97,6 +98,8 @@ async def process(self):
             await self._rooms(space=True)
         elif self.command.startswith("users"):
             await self._users()
+        elif self.command.startswith("keys"):
```

Comment:
Can we make this `pindora` given keys is a bit generic. We may want many kinds of keys, not just pindora.